src/exasol/schemas/__init__.py

```diff
@@ -0,0 +1,173 @@
+from __future__ import annotations
+
+import contextlib
+import io
+import logging
+import shutil
+from collections import defaultdict
+from dataclasses import dataclass
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from typing import Type
+
+import jinja2
+import json_schema_for_humans as json2schema
+import typer
+from json_schema_for_humans import generate
+from packaging.version import Version
+from rich.logging import RichHandler
+
+_FORMAT = "%(message)s"
+logging.basicConfig(
+    level="NOTSET",
+    format="%(message)s",
+    handlers=[RichHandler(show_time=False, show_path=False)],
+)
```

Comment:
```suggestion
_FORMAT = "%(message)s"
logging.basicConfig(
    level="NOTSET",
    format=_FORMAT,
    handlers=[RichHandler(show_time=False, show_path=False)],
)
```
❓

Reply:
Good catch, I'll remove the constant. It is a left over I decided to inline everything.